docs/source/sysadmin_guide/configure_sgx.rst

```diff
@@ -0,0 +1,458 @@
+    :ref:`val-config` below.
+
+Create a genesis batch:
+
+.. code-block:: console
+
+    $ sawtooth config genesis --key /etc/sawtooth/keys/validator.priv -o config-genesis.batch
+
+Create and submit a proposal:
+
+.. code-block:: console
+
+    $ sawtooth config proposal create -k /etc/sawtooth/keys/validator.priv \
+    sawtooth.consensus.algorithm=poet \
+    sawtooth.poet.report_public_key_pem="$(cat /etc/sawtooth/ias_rk_pub.pem)" \
+    sawtooth.poet.valid_enclave_measurements=$(poet enclave --enclave-module sgx measurement) \
+    sawtooth.poet.valid_enclave_basenames=$(poet enclave --enclave-module sgx basename) \
+    sawtooth.poet.enclave_module_name=poet_enclave_sgx.poet_enclave \
+    -o config.batch
+
+You should see a couple lines of output showing that the SGX enclave has been initialized:
+
+.. code-block:: console
+
+    [12:03:58 WARNING poet_enclave] SGX PoET enclave initialized.
+    [12:03:59 WARNING poet_enclave] SGX PoET enclave initialized.
+
+There’s quite a bit going on in the previous command, so let’s take a closer look at what it accomplishes:
+
+1.  ``sawtooth.consensus.algorithm=poet``
+    Changes consensus algorithm to PoET 
+
+2.  ``sawtooth.poet.report_public_key_pem="$(cat /etc/sawtooth/ias_rk_pub.pem)"``
+    Adds the public key that the validator registry transaction processor uses
+    to verify attestation reports.
+
+3.  ``sawtooth.poet.valid_enclave_measurements=$(poet enclave --enclave-module sgx measurement)``
+    Adds the enclave measurement for your enclave to the blockchain so that the
+    validator registry transaction processor has something it can use to check
+    signup information.
+
+4.  ``sawtooth.poet.valid_enclave_basenames=$(poet enclave --enclave-module sgx basename)``
+    Adds the enclave basename for your enclave to the blockchain so that the
+    validator registry transaction processor has something it can use to check
+    signup information.
+
+5.  ``sawtooth.poet.enclave_module_name`` is the name of the Python module that
+    implements the PoET enclave.  In this case ``poet_enclave_sgx.poet_enclave``
+    is the SGX version of the enclave - including the Python code as well as
+    the Python extension.
+
+Create a poet-genesis batch:
+
+.. code-block:: console
+
+    $ poet genesis -k /etc/sawtooth/keys/validator.priv --enclave-module sgx -o poet_genesis.batch
+    Writing key state for PoET public key: 0387a451...9932a998
+    Generating poet_genesis.batch
+
+Create genesis block:
+
+.. code-block:: console
+
+    $ sawtooth admin genesis config-genesis.batch config.batch poet_genesis.batch
+
+You’ll see some output indicating success:
+
+.. code-block:: console
+
+    Processing config-genesis.batch...
+    Processing config.batch...
+    Processing poet_genesis.batch...
+    Generating /var/lib/sawtooth/genesis.batch
+
+Genesis configuration is complete! Log out of the sawtooth account:
+
+.. code-block:: console
+
+    $ exit
+
+
+.. _val-config:
+
+Validator config file
```

Comment:
Start with a verb (to match the other subheads in this document):

    Change the validator config file